src/modules/jcms_rest/src/Plugin/rest/resource/AbstractRestResourceBase.php

```diff
@@ -582,17 +478,22 @@ public function getEntityQueueItem(EntityInterface $node, FieldItemListInterface
     $item_values = [
       'title' => $node->getTitle(),
       'image' => $this->processFieldImage($node->get('field_image'), TRUE, 'banner', TRUE),
-      'item' => [],
     ];
 
     if ($related->getType() == 'article') {
-      $item_values['item'] = $this->getArticleSnippet($related);
+      if ($article = $this->getArticleSnippet($related)) {
+        $item_values['item'] = $article;
```

Comment:
This and similar changes help us cope with the fact that some article json is still invalid.